### PR TITLE
add apiRequest method

### PR DESCRIPTION
**Pull request by ksvirkou-hubspot**
https://github.com/HubSpot/hubspot-api-nodejs/issues/152

### CODE REVIEW

File: src/services/Request.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    apiKey: 'qs',
+    accessToken: 'oauth2',
+    developerApiKey: 'qs',
+  }
+
+  constructor(config: IConfiguration = {}) {
+    this.config = config
+  }
+
+  public async sent(opts: any) {
```
Comment (atanasiuk-hubspot) [inline src/services/Request.ts:19]: Did you mean `send`?

Comment (atanasiuk-hubspot) [inline src/services/Request.ts:19]: I think you should extract this method from `Request` creating a separate `HttpClient` or similar class

**Comment by robotshatemonkeys** [comment]
Line 48     
`return await respose.json()`
fails when response Returns a 204 No Content on success instead of a JSON.

**Comment by ksvirkou-hubspot** [comment]
I'll fix it thank you

**Pull request merged**